src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -852,7 +852,7 @@ internal ImmutableArray<TypeWithAnnotations> ParameterTypesWithAnnotations
         internal TypeSymbol GetParameterType(int index) => ParameterTypesWithAnnotations[index].Type;
 
         /// <summary>
-        /// Null if no parameter is ref/out. Otherwise the RefKind for each parameter.
+        /// default if no parameter is ref/out. Otherwise the RefKind for each parameter.
```

Comment:
I got burnt by checking this for null instead of IsDefault so changed the documentation